controllers/ProposalController.php

```diff
@@ -570,4 +572,91 @@ private function removeExistingUploadedFile(string $filepath)
             throw new CannotDeleteFileException();
         }
     }
+
+
+    public function actionPostComment()
+    {
+        $model = new ManageCommentForm();
+
+        if ($model->load(Yii::$app->request->post()) && $model->validate()) {
+            $referrer = explode('/', Yii::$app->request->referrer);
+            $proposalId = $referrer[count($referrer) - 1];
```

Comment:
I think it could be better (more understandable) to subit form to /proposals/post-comment/<proposal id> or just have a hidden input with the proposal ID. In all cases, you have to check if the user who is posting a comment is allowed to do it.